advice/advice.html

```diff
@@ -20,36 +20,43 @@ <h1 class="covid-19">C<img src="../assets/coviddd.png" class="covid-img">vid-19<
         <img src="../assets/dark-mode.png" class="dark-mode-icon" id="icon">
     </header>
     <section class="container">
-        <article class="part-1">
+        <artical class="cont">
+
             <section class="part">
-                <img src="../assets/01.png" class="img-adv">
+                <img src="../images/01.png" class="img-adv">
```

Comment:
Why you changed the folder from assets to images? Are you sure that you have images folder in your branch?
Are you sure that you solved the conflict in a right way?